x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elser/ElserMlNodeService.java

```diff
@@ -243,6 +245,46 @@ public void infer(
         );
     }
 
+    @Override
+    public void chunkedInfer(
+        Model model,
+        List<String> input,
+        Map<String, Object> taskSettings,
+        InputType inputType,
+        ChunkingOptions chunkingOptions,
+        ActionListener<ChunkedInferenceServiceResults> listener
+    ) {
+        try {
+            checkCompatibleTaskType(model.getConfigurations().getTaskType());
+        } catch (Exception e) {
+            listener.onFailure(e);
```

Comment:
Should we return immediately after this?

Reply:
😨 good catch